modules/apps/web-experience/product-navigation/product-navigation-product-menu-theme-contributor/src/main/resources/META-INF/resources/product_navigation_product_menu.scss

```diff
@@ -127,6 +127,10 @@
 			padding: 10px 40px 10px 24px;
 		}
 
+		.aspect-ratio-bg-cover {
+			background-size: contain;
```

Comment:
Hey @natecavanaugh 
.aspect-ratio-bg-cover is in Lexicon and we could create a new .aspect-ratio-bg-contain class, but we are running out the time. I really dislike this temporal solution, if you have better idea, please we can go ahead with it